src/config/dark-sites.config

```diff
@@ -576,6 +577,7 @@ igg-games.com
 iipython.dev
 imageglass.org
 imgbox.com
+imgs.pandadev.net
```

Comment:
With disabled Dark Reader I opened the page and saw this:
![obraz](https://github.com/user-attachments/assets/9816e470-584e-451d-87df-61dabf7ac161)

That disqualifies the site to add here.